Tools/Scripts/webkitpy/w3c/test_exporter.py

```diff
@@ -80,9 +80,12 @@ def __init__(self, host, options, gitClass=Git, bugzillaClass=Bugzilla, WPTGitHu
         self._linter = WPTLinterClass(self._options.repository_directory, host.filesystem)
 
         self._bugzilla_url = "https://bugs.webkit.org/show_bug.cgi?id=" + str(self._bug_id)
+        if self._bug_id:
+            self._title = self._bugzilla.fetch_bug_dictionary(self._bug_id)["title"].replace("[", "\\[").replace("]", "\\]")
         self._commit_message = options.message
         if not self._commit_message:
-            self._commit_message = 'WebKit export of ' + self._bugzilla_url if self._bug_id else 'Export made from a WebKit repository'
+            self._commit_message = 'WebKit export: ' + self._title if self._title else 'Export made from a WebKit repository'
+            self._commit_message += '\r\n\r\n' + self._bugzilla_url
```

Comment:
I think you can drop `\r` here.

Reply:
The other browsers all use git trailers for URLs, so probably something like:

```suggestion
            self._commit_message += '\n\nWebKit-Bugzilla: ' + self._bugzilla_url
```

…or something?

And yeah, nothing else uses CRLF for git commits.